implementations/rust/ockam/ockam_command/src/identity/create.rs

```diff
@@ -26,11 +28,18 @@ pub struct CreateCommand {
     /// Vault name to store the identity key
     #[arg(long, value_name = "VAULT_NAME", global = true)]
     vault: Option<String>,
+
+    #[arg(short, long)]
+    key_id: Option<String>,
```

Comment:
This should be renamed to `kms_key_id`. You can use the `long = "key-id"` to keep the argument name.